concent_api/core/tests/test_auth_get_task_result.py

```diff
@@ -7,25 +7,18 @@
 from golem_messages         import message
 from golem_messages.message.concents import FileTransferToken
 
-from core.models            import PendingResponse
-from core.models            import Subtask
-from core.tests.utils       import ConcentIntegrationTestCase
+from common.constants import ErrorCode
+from common.testing_helpers import generate_ecc_key_pair
+from conductor.service import result_upload_finished
+from core.models import PendingResponse
+from core.models import Subtask
+from core.tests.utils import ConcentIntegrationTestCase
 from core.tests.utils import parse_iso_date_to_timestamp
-from common.constants        import ErrorCode
-from common.testing_helpers  import generate_ecc_key_pair
 
 
 (CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
```

Comment:
Please change style here to our actual convention

Reply:
:+1: done